.github/NEW_IA_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# New {IA Name} Fathead Instant Answer
+Help DuckDuckGo become the best Search Engine for {Language} developers!
```

Comment:
Maybe add "by creating an Instant Answer for {Language} documentation" or similar?